src/components/study-container.js

```diff
@@ -644,8 +654,6 @@ export function StudyContainer({ view, onChangeTab }) {
             const ws = connectNotifications(studyUuid);
             const wsDirectory = connectDeletedStudyNotifications(studyUuid);
 
-            loadNetworkRef.current();
```

Comment:
This loadNetwork() is executed when the current node is not built. So it does nothing.

Reply:
Here we want to force the network reload when we get the websocket connection back after a network problem maybe ?

Reply:
No this code is not executed in this case. It maybe just be cruft that accumulated over time and can ben safely removed